pyfr/solvers/euler/inters.py

```diff
@@ -4,10 +4,35 @@
                                     BaseAdvectionMPIInters,
                                     BaseAdvectionBCInters)
 
+class FluidIntIntersMixin:
+    def __init__(self, be, lhs, rhs, elemap, cfg):
```

Comment:
If you can make the args `(self, *kargs, **kwargs)` and pass them as such.  Avoids explicitly depending on what they are.